DOCKER.md

```diff
@@ -101,6 +101,40 @@ docker run hello-world
 docker compose version
 ```
 
+## Running a fully Docker Serup
+
+These steps will use Docker Compose to run everything entirely within Docker. This is the easiest
+path since it relegates all responsibility for the development environment to Docker.
+
+If you want to run a native server and just use Docker to handle the services, such as the database,
+then skip ahead to the next section and all other sections referring to the "Native Dashboard Server".
+
+For the fully contained setup, you need to build the Dashboard code-dot-org Docker image:
+
+```shell
+FIXUID=$(id -u) docker compose build web test selenium-video
```

Comment:
So that the docker container user has the same id as the host user, so when it creates files the host user owns them. you do this on the build for the image so that you do not need root to run the image.